packages/core-components/src/lib/molecules/sql-console/SqlConsole.svelte

```diff
@@ -0,0 +1,135 @@
+<script context="module">
+	export const evidenceInclude = true;
+</script>
+
+<script>
+	/** @typedef {import("@evidence-dev/query-store").QueryStore} QueryStore */
+	/** @typedef {import("./sqlConsole.action.js").SqlConsoleArgs} SqlConsoleArgs */
+	import { sqlConsole, buildAutoCompletes } from './sqlConsole.action.js';
+
+	import { onMount } from 'svelte';
+	import { slide } from 'svelte/transition';
+
+	import { Button } from '../../atoms/button';
+	import DataTable from '../../unsorted/viz/table/DataTable.svelte';
+
+	import { Eye, EyeOff, PlayerPlay } from '@evidence-dev/component-utilities/icons';
+	import { buildQuery } from '@evidence-dev/component-utilities/buildQuery';
+
+	/** @type {boolean} */
+	export let hideErrors = false;
+	/** @type {string} */
+	export let initialQuery = '';
+	/** @type {boolean} */
+	export let showResults = true;
+	/** @type {boolean} */
+	export let disabled = false;
+
+	/** @type {string} */
+	let currentQuery = initialQuery;
+
+	/** @type {string} */
+	let editorQuery = currentQuery;
+
+	/*
+	 - Try to fix autocomplete
+	 	- non-namespaced?
+		- keyword updates
+	 */
+
+	/** @type {QueryStore} */
+	export let data = buildQuery(currentQuery);
+
+	$: if (currentQuery) {
+		data = buildQuery(currentQuery);
```

Comment:
It would be nice to have something that strips semicolons, it tripped me up for a second